### ISSUE TITLE
Addition of contact info to the templates

### ISSUE THREAD

**Issue by praneeth-rdy** [Praneeth-rdy/cvGenerator#6]
## Type of Issue
- [x] Feature
- [ ] Bug
- [ ] Documentation

#### This issue requires work on the following:
1. Addition of contact info to the CV templates would be an enhancement
2. Contact info like LinkedIn, GitHub, Mobile and Email would suffice.

**Comment by kwennB** [comment]
Hello @Praneeth-rdy 
Will this enhancement be added to the template or the root index.html?

**Comment by praneeth-rdy** [comment]
Hi @kwennB 

This enhancement would be added to the CV templates to help users add their contact information to the CVs.

**Sample**
![cv-contact-info-sample](https://user-images.githubusercontent.com/57290210/172997184-d51097df-faad-45ff-b2d1-4bef23703b3f.png)

**Comment by kwennB** [comment]
> Hi @kwennB 
> 
> 
> 
> This enhancement would be added to the CV templates to help users add their contact information to the CVs.
> 
> 
> 
> **Sample**
> 
> ![cv-contact-info-sample](https://user-images.githubusercontent.com/57290210/172997184-d51097df-faad-45ff-b2d1-4bef23703b3f.png)
> 
> 
> 
> 

Okay that works. The contact info will be in the header (like a navigation) 
HTML, CSS would suffice?

**Comment by praneeth-rdy** [comment]
Yup. HTML and CSS would suffice. You can utilise the extra space available in the header of the CV template (below the institute name) as shown below.

![Screenshot from 2022-06-10 17-13-54](https://user-images.githubusercontent.com/57290210/173057859-cac2f893-dd75-412d-8c67-d554f052f93b.png)

**Comment by kwennB** [comment]
> Yup. HTML and CSS would suffice. You can utilise the extra space available in the header of the CV template (below the institute name) as shown below.
> 
> 
> 
> ![Screenshot from 2022-06-10 17-13-54](https://user-images.githubusercontent.com/57290210/173057859-cac2f893-dd75-412d-8c67-d554f052f93b.png)
> 
> 
> 
> 

Okay will do, I will make a PR as soon as I can. (Before the end of next week)

**Comment by praneeth-rdy** [comment]
Cool. I'm assigning you to this issue. Once you complete the development, create a PR and request a review.
